lib/msal-core/src/AuthenticationRequestParameters.ts

```diff
@@ -71,6 +72,8 @@ export class AuthenticationRequestParameters {
             authEndpoint += "&";
         }
         let requestUrl: string = `${authEndpoint}${str.join("&")}`;
+        //Library already handles prompt values, remove duplicate prompt values to avoid server error.
+        requestUrl = Utils.urlRemoveQueryStringParameter(requestUrl, Constants.prompt);
```

Comment:
We should remove this code. We can't remove prompt, Neha has another PR to deal with prompt.

Reply:
we will have to selectively remove prompt

Reply:
Remove prompt from ATS calls.